packages/validation/README.md

```diff
@@ -0,0 +1,11 @@
+# Validation
+
+A package of technical, and business rule validation logic.
+
+This should be required as a file dependency as will not be published to an
+npm repository. To use it, add this line to your `package.json` `dependencies`
+and run`npm install`.
+
+```
+"@pins/validation": "file:../validation",
+```
```

Comment:
I extracted out the validations to be sharable... but now I wonder if this needs to happen, if we only validate inside the appeals service API ?